tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -188,3 +230,45 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
 		}, nil),
 	)
 })
+
+// Temporary VM-specific copy of runRestore. This is here to avoid making big
+// changes to runRestore when they are likely to be taken out in the near future.
+func runVmRestore(brCase VmBackupRestoreCase, backupName, restoreName string, nsRequiresResticDCWorkaround bool) {
+	log.Printf("Creating restore %s for case %s", restoreName, brCase.Name)
+	restore, err := lib.CreateRestoreFromBackup(dpaCR.Client, namespace, backupName, restoreName)
+	Expect(err).ToNot(HaveOccurred())
+	Eventually(lib.IsRestoreDone(dpaCR.Client, namespace, restoreName), timeoutMultiplier*time.Minute*60, time.Second*10).Should(BeTrue())
+	// TODO only log on fail?
+	describeRestore := lib.DescribeRestore(veleroClientForSuiteRun, dpaCR.Client, restore)
+	GinkgoWriter.Println(describeRestore)
+
+	restoreLogs := lib.RestoreLogs(kubernetesClientForSuiteRun, dpaCR.Client, restore)
+	restoreErrorLogs := lib.RestoreErrorLogs(kubernetesClientForSuiteRun, dpaCR.Client, restore)
+	accumulatedTestLogs = append(accumulatedTestLogs, describeRestore, restoreLogs)
+
+	if !brCase.SkipVerifyLogs {
+		Expect(restoreErrorLogs).Should(Equal([]string{}))
+	}
+
+	// Check if restore succeeded
+	succeeded, err := lib.IsRestoreCompletedSuccessfully(kubernetesClientForSuiteRun, dpaCR.Client, namespace, restoreName)
+	if brCase.RestoreErr != nil {
+		Expect(succeeded).To(Equal(false))
```

Comment:
from CI logs, seems restore is working :eyes:

Reply:
this PR was created after rebase, so it is using new openshift velero image. Could it be making tests pass?

Reply:
I think that must be the case, I didn't realize it was fixed already.

Reply:
just to make sure, this commit fixed the issue? https://github.com/openshift/velero/commit/7cb5b960190a7cb4cc490b3e5b0f72bf222a7f0f

Reply:
I am pretty sure that's right, maybe @shubham-pampattiwar can confirm?

Reply:
yes restored PVCs with pending state wont partially fail the restore